pkg/api/aim/query/query_test.go

```diff
@@ -32,36 +34,132 @@ func (s *QueryTestSuite) SetupTest() {
 	s.db = db
 }
 
-func (s *QueryTestSuite) Test_Ok() {
+func (s *QueryTestSuite) TestPostgresDialector_Ok() {
 	tests := []struct {
 		name         string
 		query        string
+		dialector    string
```

Comment:
Since this test is entirely posgres dialector, this parameter could be defaulted

Reply:
yep, removed.